ui/ops/src/traits/airQuality/airQuality.js

```diff
@@ -150,3 +151,27 @@ export function useAirQuality(value) {
     tableData
   };
 }
+
+/**
+ * @param {MaybeRefOrGetter<string|ListAirQualityHistoryRequest.AsObject>} req - The request
+ * @param {MaybeRefOrGetter<boolean>=} paused - Whether to pause the data stream
+ * @return {Promise<ListAirQualityHistoryResponse.AsObject>}
+ */
+export function useListAirQualityHistory(req, paused = false) {
+  const tracker = reactive(
+      /** @type {ActionTracker<AirQualityRecord.AsObject>} */
+      newActionTracker()
+  );
+  return listAirQualitySensorHistory(req, tracker);
+}
+
+/**
+ * @param {AirQualityRecord | AirQualityRecord.AsObject} obj
+ * @return {AirQualityRecord.AsObject & {recordTime: Date|undefined}}
+ */
+export function airQualityRecordToObject(obj) {
+  if (!obj) return undefined;
+  if (typeof obj.toObject === 'function') obj = obj.toObject();
+  if (obj.recordTime) obj.recordTime = timestampToDate(obj.recordTime);
+  return obj;
+}
```

Comment:
I think this deserves to live in `src/api/...` instead of here as it's more useful than these components.